From 9719647b5268a9f1dc744c4373c6ba71a99622e0 Mon Sep 17 00:00:00 2001
From: Paul Pfeister <code@pfeister.dev>
Date: Tue, 2 Jul 2024 02:16:03 -0400
Subject: [PATCH] Downgrade anstream

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 166af77..233b18e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,7 +24,7 @@ futures = "0.3"
 rlimit = "0.10.1"
 log = "0.4.0"
 env_logger = "0.11.3"
-anstream = "=0.6.14"
+anstream = "=0.6.13"
 dirs = "5.0.1"
 gcd = "2.0.1"
 rand = "0.8.5"
-- 
2.45.1

